R/standalone-check_pkg_installed.R

```diff
@@ -10,7 +10,6 @@
 # This file provides functions to check package installation.
 #
 # ## Changelog
-# nocov start
```

Comment:
The reason I wrapped all the standalone files in the `# nocov start` is so when we import these scripts into other pkgs they don't count against the package's code coverage if we don't use every check in that project.

It would be awesome to have code coverage for the standalone package, but I am not sure the best way to do it. Two things come to mind:

1. Do nothing and we don't have code coverage for the standalone pkg.
2. In the test coverage workflow, we could import every R script, remove `# nocov start`, and write it back to disc. 

If we opt for doing number 2, we can tackle that in a separate PR. Is there something else you can think of?